earth_enterprise/src/fusion/autoingest/sysman/gefusion

```diff
@@ -105,6 +129,22 @@ stop() {
   fi
 }
 
+status() {
+  if check_if_master ; then
+	check_daemon gesystemmanager
```

Comment:
Minor: Fix indentation.

Reply:
Yes I shouldnt mix tabs and spaces, thanks.